sdk/ai/azure-ai-projects/README.md

```diff
@@ -327,6 +330,95 @@ agent = project_client.agents.create_agent(
 
 <!-- END SNIPPET -->
 
+#### Create Agent with Enterprise File Search
+
+We can upload file to Azure as it is shown in the example, or use the existing Blob. In the code below we demonstrate how this can be achieved. First we are upload file to azure and create `VectorStoreDataSource`, which then is used to create vector store. This vector sore is then given to the `FileSearchTool` constructor.
```

Comment:
I suggest "Azure blob storage" instead of just "Blob"

Reply:
Fixed.